server/src/catalog.rs

```diff
@@ -17,6 +17,9 @@ use std::io::Read;
 use std::sync::Arc;
 use std::{fs::File, path::Path};
 use std::str;
+use std::fs::OpenOptions;
+
+use tracing::{debug, error, info, warn, Level, event, instrument, span};
```

Comment:
This will potentially be handled automatically by rustfmt in the future but for now we group imports manually as std -> external crates -> internal crates. The tracing import should be moved to the same group as datafusion (which should be one group).

Reply:
It looks like std -> external crates -> internal crates to me?

Reply:
Fixed